tests/integration/targets/test_create_rds_global_cluster/tasks/main.yml

```diff
@@ -0,0 +1,87 @@
+        create_rds_global_cluster_instance_class: "{{ test_instance_class }}"
+        create_rds_global_cluster_master_username: "{{ test_username }}"
+        create_rds_global_cluster_master_user_password: "{{ test_password }}"
+        create_rds_global_cluster_global_cluster_name: "{{ test_global_cluster_name }}"
+        create_rds_global_cluster_primary_cluster_name: "{{ test_primary_cluster_name }}"
+        create_rds_global_cluster_primary_cluster_region: "{{ test_primary_cluster_region }}"
+        create_rds_global_cluster_primary_cluster_instance_name: "{{ test_primary_cluster_instance_name }}"
+        create_rds_global_cluster_replica_cluster_name: "{{ test_replica_cluster_name }}"
+        create_rds_global_cluster_replica_cluster_region: "{{ test_replica_cluster_region }}"
+        create_rds_global_cluster_replica_cluster_instance_name: "{{ test_replica_cluster_instance_name }}"
+        create_rds_global_cluster_replica_enable_global_write_forwarding: "{{ test_replica_enable_global_write_forwarding }}"
+
+    - name: Get Global DB information
+      amazon.cloud.rds_global_cluster:
+        global_cluster_identifier: "{{ test_global_cluster_name }}"
+        region: "{{ test_primary_cluster_region }}"
+        state: describe
+      register: global_cluster_info
+
+    - name: Get primary DB cluster information
+      amazon.aws.rds_cluster_info:
+        cluster_id: "{{ test_primary_cluster_name }}"
+        region: "{{ test_primary_cluster_region }}"
+      register: primary_cluster_info
+
+    - name: Get secondary DB cluster information
+      amazon.aws.rds_cluster_info:
+        cluster_id: "{{ test_replica_cluster_name }}"
+        region: "{{ test_replica_cluster_region }}"
+      register: replica_cluster_info
+
+    - name: Verify resources are created successfully
+      assert:
+        that:
+          - global_cluster_info.result.identifier == "{{ test_global_cluster_name }}"
+          - global_cluster_info.result.properties.source_db_cluster_identifier == primary_cluster_info.clusters[0].db_cluster_arn
+          - primary_cluster_info.clusters[0].db_cluster_identifier == "{{ test_primary_cluster_name }}"
+          - "'{{ test_primary_cluster_region }}' in primary_cluster_info.clusters[0].availability_zones[0]"
+          - replica_cluster_info.clusters[0].db_cluster_identifier == "{{ test_replica_cluster_name }}"
+          - "'{{ test_replica_cluster_region }}' in replica_cluster_info.clusters[0].availability_zones[0]"
+          - replica_cluster_info.clusters[0].replication_source_identifier == primary_cluster_info.clusters[0].db_cluster_arn
+
+  always:
+
+    - name: Delete global db and other role infrastructure
+      ansible.builtin.include_role:
+        name: cloud.aws_ops.create_rds_global_cluster
+      vars:
+        create_rds_global_cluster_operation: delete
+        create_rds_global_cluster_engine: "{{ test_engine }}"
```

Comment:
You can remove the vars that are not needed for delete operation.

Reply:
removed